libs/ledger-live-common/src/bot/cli.ts

```diff
@@ -0,0 +1,18 @@
+import "../__tests__/test-helpers/environment";
+import { bot } from ".";
+
+const { BOT_FILTER_CURRENCY, BOT_FILTER_FAMILY } = process.env;
+const arg: {
+  currency?: string;
+  family?: string;
+} = {};
+
+if (BOT_FILTER_CURRENCY) {
+  arg.currency = BOT_FILTER_CURRENCY;
```

Comment:
Just an idea/improvement (or maybe it's already supported) but could we filter on multiple currencies/families ?

Reply:
yeah if there is a need we could.